README.md

```diff
@@ -47,7 +47,7 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 ## Supported devices
 ### lk2nd-msm8909
 - Acer Liquid Z330 - T01
-- Mobvoi TicWatch Pro - catfish
+- Mobvoi TicWatch Pro - catfish (use `lk2nd-msm8909-appended-dtb.img`)
```

Comment:
Only a comment but this should work on all of the following codenames: `catfish, catshark, catfish_ext` with lk2nd-msm8909-appended-dtb.img (WIP for lk1st by me and @wonderfulShrineMaidenOfParadise in IRC) 

NOTE: EDL cable is not optional but REQUIRED, as well as the micro-usb to TicWatch adapter that usually saus "Watch Pro" on the back (not my typo), else you will be responsible.

Reply:
Current dts: https://github.com/Nexusma/msm-catfish-9.0-wear-os/blob/master/arch/arm/boot/dts/qcom/catfish.dts

I cannot promise for catshark, as the dts for it is not added here:
https://github.com/Nexusma/msm-catfish-9.0-wear-os/blob/master/arch/arm/boot/dts/qcom/catshark.dts